src/ipa/rpi/controller/rpi/sync.cpp

```diff
@@ -0,0 +1,349 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2024, Raspberry Pi Ltd
+ *
+ * sync.cpp - sync algorithm
+ */
+#include "sync.h"
+
+#include <chrono>
+#include <ctype.h>
+#include <fcntl.h>
+#include <list>
+#include <map>
+#include <strings.h>
+#include <unistd.h>
+#include <vector>
```

Comment:
fnctl, list, map and possibly vector not needed?

Reply:
Indeed, vector, map and list all superfluous.